media/css/firefox/family/components/modules/_faq.scss

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@use 'sass:color';
+
+@use '../../utils' as f3;
```

Comment:
Not a blocker: Why as `f3` and not as `utils`?

Reply:
Personal preference that stuck. From a folder view, `utils` filename works for me because it clearly belongs in the `firefox/family` folder, but from a single file view, `utils.$border-width` felt too generic as a namespace (could be utils from bedrock or firefox). So I wanted to call out that these variables and mixins are specifically from the Firefox for Families (F3) design.

Reply:
Ah, cool!